layouts/release/section.html

```diff
@@ -1,33 +1,215 @@
+            {{ if (eq (len (findRE `(\.[^\/]{3,4}|\/)$` $normalized_cta_url)) 0) }}
+              {{ $normalized_cta_url = printf "%s/" $normalized_cta_url }}
+            {{ end }}
+            
+            {{ $abs_cfa := printf "%s" ($normalized_cta_url | absURL) }}
+            {{ $is_cfa_page := 0 }}
+            {{ if or (eq $normalized_cta_url .Permalink) (eq $abs_cfa .Permalink) }}
+              {{ $is_cfa_page = 1 }}
+            {{ end }}
+            {{ if and (eq $has_cfa 1) (eq .Page.Params.show_call_for_action_button "true") }}
+              <div class="{{ $.Site.Params.call_for_action_wrapper_classes | default "float-right hidden-xs" }}" id="btn-call-for-action">
+                <div id="btn-call-for-action"><a href="{{ .Site.Params.call_for_action_url }}" class="{{ .Site.Params.call_for_action_classes | default "btn btn-huge btn-warning" }}"><i class="fa {{ .Site.Params.call_for_action_icon }}"></i> {{ .Site.Params.call_for_action_text }}</a></div>
+              </div>
+            {{ end }}
+            <div class="{{ $.Site.Params.main_menu_classes | default "navbar float-sm-right yamm" }}" id="main-menu">
+              <div class="navbar-collapse collapse reset float-right" id="navbar-main-menu">
+                <ul class="nav navbar-nav">
+                  {{ partial "nav.html" . }}
+                </ul>
+              </div>
+              {{ partial "nav_toggle.html" . }}
+            </div>
           </div>
         </div>
       </div>
     </div>
+    <!---Navbar end --->
+
+    {{ if isset .Page.Params "headline" }}
+    <div class="jumbotron featured-jumbotron featured-jumbotron-download vertical-align flex-column margin-bottom-0">
+
+      <div class="flex-grow vertical-align-space-evenly flex-wrap">
+        <div class="col-md-10 flex-column">
+          <p style="align-self: flex-start;"><span class="p text-orange">version 9</span> new release</p>
+          <h1>Jakarta&nbsp;EE</h1>
+        </div>
+        <div class="col-md-2 col-md-offset-2 vertical-align">
+          <span class="jumbotron-right-content"><i>9</i></span>
+        </div>
+        <div class="col-md-9 col-md-offset-1 vertical-align" style="text-align: left;">
+          <p><span class="p text-orange">An open, community-driven platform</span> <br> for future innovation of <br> enterprise Java technologies</p>
+        </div>
+      </div>
+
+      <div class="container jumbotron-button-container">
+        <ul class="list-inline buttons">
+          <li class="vcenter"><a class="btn btn-primary vertical-align" href="#">DOWNLOAD <br/> COMPATIBLE PRODUCTS</a></li>
+          <li class="vcenter"><a class="btn btn-primary vertical-align" href="#">SPECIFICATIONS</a></li>
+        </ul>
+      </div>
+    </div>
     {{ end }}
   </header>
+{{ end }}
+
+{{ define "footer" }}
+<p id="back-to-top">
+  <a class="visible-xs" href="#top">{{ i18n "navigation-back-to-top" }}</a>
+</p>
+<footer id="footer-release-9" class="footer-release-9">
```

Comment:
Do you need to recreate the footer here?

Reply:
This is because I cannot customize the style sue to the template is using "id", which stops my custom style. And before there was one more column, but after discussion, we decide to remove it. 

But for the customize footer style, I created PR https://github.com/EclipseFdn/hugo-solstice-theme/pull/179 and https://github.com/EclipseFdn/solstice-assets/pull/184 . I think if they are merged, we can add param to custom footer. What do you think? 🤔

Reply:
Yes I think this would be a good solution and it would prevent us from hardcoding the footer here.